### PR TITLE
Added response to address balance

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**

### CODE REVIEW

File: src/api/routes/btc.ts [hunk 1]
```diff
@@ -38,11 +38,30 @@ export const BtcRoutes: FastifyPluginCallback<
     schema: {
       params: Type.Object({
         address: Type.String({
-          description: 'Specify either a Stacks or Bitcoin address',
+          description: 'Specify either an STX(Stacks) or btc () Bitcoin address',
```
Comment (rafaelcr) [inline src/api/routes/btc.ts]:
```suggestion
          description: 'Specify either an STX(Stacks) or BTC(Bitcoin) address',
```

File: src/api/routes/btc.ts [hunk 2]
```diff
@@ -38,11 +38,30 @@ export const BtcRoutes: FastifyPluginCallback<
...
       })
+    },
+    response: {
+      200: Type.Object({
+        description: "success response",
+        properties: {
+          stacks: Type.Object({
+            address: Type.String({
+              description: 'Specify either an STX(Stacks) or btc () Bitcoin address',
+              examples: ['SPRSDSRT18DS9R8Y2W13JTKF89NFHEGDWQPB78RE', '15XCtJkEDxE1nhFawvPY4QEkEyNywxNSfL'],
```
Comment (rafaelcr) [inline src/api/routes/btc.ts]: This should say this is for a Stacks address only (see other comment)

File: src/api/routes/btc.ts [hunk 3]
```diff
@@ -38,11 +38,30 @@ export const BtcRoutes: FastifyPluginCallback<
...
+            }),
+            balance: Type.Integer({
+              description: 'Balance for the stacks or btc address',
+              examples: ['0']
+            })
+          })
```
Comment (rafaelcr) [inline src/api/routes/btc.ts]: You'll still need to add the `bitcoin` response

File: src/api/routes/btc.ts [hunk 4]
```diff
@@ -36,13 +36,54 @@ export const BtcRoutes: FastifyPluginCallback<
...
       params: Type.Object({
         address: Type.String({
-          description: 'Specify either a Stacks or Bitcoin address',
+          description: 'Specify either an STX(Stacks) or BTC(Bitcoin) address. The below example takes STX address as input.',
           examples: ['SPRSDSRT18DS9R8Y2W13JTKF89NFHEGDWQPB78RE', '15XCtJkEDxE1nhFawvPY4QEkEyNywxNSfL'],
         }),
-      })
+      }),
+      response: {
+        200: Type.String({
```
Comment (rafaelcr) [inline src/api/routes/btc.ts:51]: This should be `Type.Object`

**Comment by zone117x** [comment]
Thanks @LakshmiLavanyaKasturi! Closing since you completed in https://github.com/rafaelcr/stx-btc-api/pull/20

**Pull request closed without merging**